src/Pages/Login.cshtml.cs

```diff
@@ -30,10 +31,10 @@ public class LoginModel : PageModel
 
         private UserDatabase UserDatabase { get; }
         private EmailSender EmailSender { get; }
-        public LoginModel(UserDatabase userDatabase, EmailSender emailSender)
+        public LoginModel(UserDatabase userDatabase, IOptions<EmailSender> emailSenderSettings)
         {
             UserDatabase = userDatabase;
-            EmailSender = emailSender;
+            EmailSender = emailSenderSettings.Value;
```

Comment:
Is this some kind of fix related that it don't works for me in dev? :D

Reply:
Yeah, I've fixed in dev now as well :D